src/Microsoft.CSharp/tests/CSharpGetIndexBinderTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.CSharp.RuntimeBinder.Tests
+{
+    public class CSharpGetIndexBinderTests
+    {
+        [Fact]
+        public void ArrayIndexing()
+        {
+            dynamic d = new[] { 0, 1, 2, 3 };
+            for (int i = 0; i != 4; ++i)
+            {
+                Assert.Equal(i, d[i]);
+            }
+        }
```

Comment:
We shouldn't duplicate them. If we really need them in both projects, we can include the source file in both, but it'd be better to just have a p2p dependency from the one test project to all of the libraries it's testing.